assets/src/edit-story/components/panels/utils/getBoundRect.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { dataPixels } from '../../../units/dimensions';
+
+function getBoundRect(list) {
+  const firstElementProperties = list[0].rotationAngle
+    ? calcRotatedObjectPositionAndSize(
+        list[0].rotationAngle,
+        list[0].x,
+        list[0].y,
+        list[0].width,
+        list[0].height
+      )
+    : list[0];
+  let startX = firstElementProperties.x;
+  let startY = firstElementProperties.y;
+  let endX = startX + firstElementProperties.width;
+  let endY = startY + firstElementProperties.height;
+
+  list.forEach((el) => {
+    const elementProperties = el.rotationAngle
+      ? calcRotatedObjectPositionAndSize(
+          el.rotationAngle,
+          el.x,
+          el.y,
+          el.width,
+          el.height
+        )
+      : el;
+    startX = Math.min(elementProperties.x, startX);
+    startY = Math.min(elementProperties.y, startY);
+    const elEndX = elementProperties.x + elementProperties.width;
```

Comment:
Please combine these expressions inside the `Math.max`'s below.

Reply:
Done